### PR TITLE
Make the landing page for Virtual Tribunals generally

### LINKED ISSUE #140: Make the virtualtribunals and nuremberg landing pages
virtualtribunals: Contains link to Spotlight, and link to /nuremberg

nuremberg: remove current content from root and add the new route. The content of this page will be what's outlined here https://github.com/sul-dlss/nta-arclight/issues/32 -- final design not confirmed.

### PR DESCRIPTION
**Pull request by jcoyne**
Move the catalog to /nuremberg

Fixes #140

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -5,10 +5,10 @@
   mount Blacklight::Engine => '/'
   mount Arclight::Engine => '/'
 
-  root to: "catalog#index"
+  root to: "virtual_tribunals#index"
   concern :searchable, Blacklight::Routes::Searchable.new
 
-  resource :catalog, only: [:index], as: 'catalog', path: '/catalog', controller: 'catalog' do
+  resource :catalog, only: [:index], as: 'catalog', path: '/nuremberg', controller: 'catalog' do
```
Comment (marlo-longley) [inline config/routes.rb:11]: I may be misreading, but I thought during our discussion we decided that the page at `/nuremberg` should essentially be an About page, basically a static page, and not the catalog#index page. The catalog would then be linked to from that About NTA page.

Comment (marlo-longley) [reply config/routes.rb:11]: Discussion here: https://github.com/sul-dlss/nta-arclight/issues/135#issuecomment-1301051934

Comment (jcoyne) [reply config/routes.rb:11]: @marlo-longley I thought that was superseded by what @ggeisler had here: https://www.figma.com/proto/2ekjawwMTunlAesPyzIgpJ/NTA-%2F-Virtual-Tribunals-strategy?node-id=1727%3A20225&scaling=min-zoom&page-id=866%3A53780&starting-point-node-id=1727%3A20225

Comment (marlo-longley) [reply config/routes.rb:11]: I see. That makes sense. I guess during discussion the idea of making sure we could search across multiple collections in VT Arclight if more are added -- does this approach lock us in the path of one catalog controller per collection?

Comment (jcoyne) [reply config/routes.rb:11]: No, no lock-in. However, if you look at page 4 of Gary's mockup. It's not clear where `/nuremberg` will route to if we add more collections to Arclight.

Comment (marlo-longley) [reply config/routes.rb:11]: OK. Just FYI I believe Gary indicated that page 1 was the relevant page / the design Tom approved.

Comment (jcoyne) [reply config/routes.rb:11]: Right. Page 4 was the future concept where we add more arclight collections.

Comment (ggeisler) [reply config/routes.rb:11]: Just to be clear, page 4 was basically just to show that if and when we add more collections, there were things to figure out. My notes say for the ArcLight instance say:

- New, more generic title needed
- Design and dev work needed to reflect multiple collections
- URL for ArcLight home needed (“collections?”)
- URL redirect strategy needs updating

I'd guess `/nuremberg` could just go directly the `/catalog/mt839rq8746` page (NTA collection page; maybe the NTA custom homepage would be unnecessary at that point) but obviously by the time we have to deal with this a lot of other things might have happened so who knows.

**Review by marlo-longley — APPROVED** [review]

**Pull request merged**